packages/react-forms/src/InputRange/InputRange.md

```diff
@@ -10,21 +10,36 @@ additionally, this component supports two values at the same time.
 The `onChange` handler in this case will return an object containing the properties
 `start` and `end`, rather than a single number.
 
-Note that in order to switch the component behavior to the double valued alternative
-you must define a `defaultValue` defining the `start` and `end` properties, or define
-its `value`.
+Rather than expecting a `value` property, this component expects a `values` one, which is an array of 1 or 2 numbers.
+If only one number is provided, the component will render a single handle, when two are provided, the component will switch
+to the two handles layout.
 
 ```js
-<InputRange
-  defaultValue={{ start: 10, end: 40 }}
-  name="double1"
-  onChange={console.log}
-/>
+<InputRange defaultValues={[10, 40]} name="double1" onChange={console.log} />
```

Comment:
`name` attribute can be `name="double-input"`.

Reply:
I provided better names, thanks.